src/utils/DateUtils.ts

```diff
@@ -202,8 +202,7 @@ export class DateUtils {
     } else if (date !== null && !isNaN(Number(date))) {
       return moment(Number(date)).toDate();
     } else if (_.isString(date)) {
-      const formats = ['YYYY/MM/DD@HH:mm:ssZ', moment.ISO_8601];
-      const dateMoment = moment(date, formats);
+      const dateMoment = moment(date, 'YYYY/MM/DD@HH:mm:ssZ');
```

Comment:
An array of formats isn't supported anymore it seems, why was the back up moment.ISO_8601? 
I guess I could check is the date is valid first and default to that format instead 🤷